prbt_hardware_support/spec.dox

```diff
@@ -54,19 +54,12 @@
  * - If the hold step fails, the system still tries to execute the halt service.
  *    (see also \refspec{hold_service_fail})
  *
- * - If the STO changes to TRUE during the execution of the stop1 process, the
- * system stops the stop1 process as soon as the currently active step is finished.
- * For example:
- *    If the state of the STO changes from TRUE -> FALSE while the drives
- *    are recovering, the controller is kept in "holding" mode and
- *    the execution of the stop trajectory is skipped.
- *
  * Please note:
  * - While the controller is holded, the controller rejects all new trajectories.
- * - The stop1 process can also be resumed by again chainging the STO to FALSE.
+ * - The stop1 process cannot be interrupted, even if the STO changes to TRUE.
```

Comment:
I think we should specify "stop1 process"...

Reply:
We would repeat ourselves because we already specified a "stop1 execution" ...
I will try to simplify everything